modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -646,6 +650,57 @@ class ConvolutionLayerImpl : public BaseConvolutionLayerImpl
         }
     };
 
+#ifdef HAVE_OPENCL
+    bool forward_ocl(std::vector<Mat*> &inputs, std::vector<Mat> &outputs, std::vector<Mat> &internals)
+    {
+        int group = inputs[0]->size[1] / blobs[0].size[1];
+
+        if (convolutionOp.empty())
+        {
+            LibDNNConvConfig config;
+            config.in_shape = {inputs[0]->size[0], inputs[0]->size[1], inputs[0]->size[2], inputs[0]->size[3]};
```

Comment:
it's another piece of C++ 11 code. @alalek, do we check anywhere that the code is C++ 98 compliant?